### PR TITLE
Add original_spec_nvr field to rpmbuild phase

### PR DESCRIPTION
**Pull request by johnbieren**

### CODE REVIEW

**Comment by johnbieren** [comment]
Okay one sec

**Comment by pypingou** [comment]
Not knowing the pipeline well, this looks fine to me, but as I said, I'm not the best person to review :)

File: config/Dockerfiles/rpmbuild/rpmbuild-test.sh [hunk 1]
```diff
@@ -18,6 +18,8 @@ git checkout ${fed_branch}
 git checkout ${fed_rev}
 # Create new branch because fedpkg wont build with detached head
 git checkout -b test_branch
+# Get current NVR
+truenvr= rpm -q --define "dist .$fed_branch" --queryformat '%{name}-%{version}-%{release}.%{arch}\n' --specfile ${fed_repo}.spec | head -n 1
```
Comment (p3ck) [inline config/Dockerfiles/rpmbuild/rpmbuild-test.sh]: you need to wrap this in $()

**Comment by arilivigni** [comment]
we need nvr and README updated as well

**Comment by johnbieren** [comment]
@arilivigni are README and the nvr stuff not showing for you in the files changed? I just pushed that a few minutes ago.. shows for me

**Comment by pypingou** [comment]
Many thanks for your help folks, much appreciated! :)

**Pull request merged**